src/plugins/crypto.c

```diff
@@ -959,7 +959,73 @@ gboolean bd_crypto_luks_resize (const gchar *luks_device, guint64 size, GError *
  * @pass_data: (array length=data_len): a passphrase for the TrueCrypt/VeraCrypt volume (may contain arbitrary binary data)
  * @data_len: length of the @pass_data buffer
  * @read_only: whether to open as read-only or not (meaning read-write)
- * @keyfiles: (array length=keyfiles_count) paths to the keyfiles for the TrueCrypt/VeraCrypt volume
+ * @error: (out): place to store error (if any)
+ *
+ * Returns: whether the @device was successfully opened or not
+ *
+ * Tech category: %BD_CRYPTO_TECH_TRUECRYPT-%BD_CRYPTO_TECH_MODE_OPEN_CLOSE
+ */
+gboolean bd_crypto_tc_open (const gchar *device, const gchar *name, const guint8* pass_data, gsize data_len, gboolean read_only, GError **error) {
```

Comment:
I think the old function should just call the new more universal one.

Reply:
Makes sense. Done in 12841cf.

Reply:
I also squahed 12841cf into 9a7f054 during the rebasing.